locale/en-US/zotero-better-bibtex.dtd

```diff
@@ -39,7 +39,7 @@ https://github.com/retorquere/zotero-better-bibtex/issue">
 <!ENTITY better-bibtex.ItemPane.citekey_column	"Citation Key">
 
 <!ENTITY better-bibtex.Preferences.auto-abbrev "Automatically abbreviate journal title if none is set explicitly">
-<!ENTITY better-bibtex.Preferences.auto-abbrev.style "Abbreviation style:">
+<!ENTITY better-bibtex.Preferences.auto-abbrev.style "Journal abbreviations">
```

Comment:
The problem is not that this string is wrong but that it's visible in Zotero. It should only be visible in Juris-M.